src/main/java/com/endava/soa_5/exception_and_io/CustomExceptions.java

```diff
@@ -0,0 +1,8 @@
+package com.endava.soa_5.exception_and_io;
+
+public class CustomExceptions extends Exception{
+    public CustomExceptions(String errorMessage){
+        super(errorMessage);
+        System.out.println(errorMessage);
```

Comment:
No need of this 'sout'. Then this exception will be thrown , you should see a message in stack trace.

Reply:
Done!

Reply:
Done!